packages/p2p/webpack.config.js

```diff
@@ -136,9 +136,9 @@ module.exports = function () {
                 react: 'react',
                 'react-dom': 'react-dom',
                 'prop-types': 'prop-types',
-                ...(is_publishing ? {} : { formik: 'formik' }),
+                ...(is_publishing ? { 'lodash.debounce': 'lodash.debounce' } : { formik: 'formik' }),
```

Comment:
I think when we publish we can't rely on the consumer to include `lodash.debounce` but for non-published builds (e.g. for Deriv.app) we know they'll include it. 

```suggestion
                ...(is_publishing ? {} : { 'lodash.debounce': 'lodash.debounce', formik: 'formik' }),
```